Source/RetrieveAndRankV1/RetrieveAndRank.swift

```diff
@@ -827,11 +854,25 @@ public class RetrieveAndRank {
             userAgent: userAgent
         )
         
+        // construct answerMetadata object
+        var json = ["answers": 10]
+        if let numberOfResults = numberOfResults {
+            json = ["answers": numberOfResults]
+        }
+        guard let answerMetadata = try? json.toJSON().serialize() else {
+            let failureReason = "Answer metadata could not be serialized to JSON."
+            let userInfo = [NSLocalizedFailureReasonErrorKey: failureReason]
+            let error = NSError(domain: domain, code: 0, userInfo: userInfo)
+            failure?(error)
+            return
+        }
+        
         // execute REST request
         Alamofire.upload(
             request,
             multipartFormData: { multipartFormData in
-                multipartFormData.appendBodyPart(fileURL: resultsFile, name: "resultsFile")
+                multipartFormData.appendBodyPart(fileURL: resultsFile, name: "answer_data")
+                multipartFormData.appendBodyPart(data: answerMetadata, name: "answer_metadata")
```

Comment:
Hmm... this looks good to me. Did you say that it always returns 10 answers regardless of the `numberOfResults` argument?